src/h3lib/lib/algos.c

```diff
@@ -791,7 +792,7 @@ void H3_EXPORT(polyfill)(const GeoPolygon* geoPolygon, int res, H3Index* out) {
     // This first part is identical to the maxPolyfillSize above.
 
     // Get the bounding boxes for the polygon and any holes
-    STACK_ARRAY_CALLOC(BBox, bboxes, geoPolygon->numHoles + 1);
+    BBox* bboxes = calloc(geoPolygon->numHoles + 1, sizeof(BBox));
```

Comment:
I don't think this one's a risk - it scales with the number of polygons, not the number of hexagons, and should be fine with stack allocation.

Reply:
I say better safe than sorry here. What if someone provides a geofence of the US, but just does it lazily by shoving in all of the county geofences in a huge array? (Or attempting to guarantee exact set matching if they need national and county level hexagon sets as a more rational explanation.)

Reply:
Point taken. I didn't realize we were removing all instances of `STACK_ARRAY_CALLOC`, in which case sure. No real preference here.